inference-test-tool/run.py

```diff
@@ -106,6 +112,9 @@ def upload_study_me(file_path, is_segmentation_model, host, port):
         print("Saving output masks to files 'output/output_masks_*.npy")
         for index, mask in enumerate(masks):
             mask.tofile('output/output_masks_{}.npy'.format(index + 1))
+    else:
```

Comment:
If it's not a segmentation model, it may not necessarily be a bbox model. Maybe we should add a -b flag (analogous to the -s flag) to tell the tool that it should processing bbox outputs.